Modules/Components/Examples/include/selxIdentityTransformRegistrationComponent.h

```diff
@@ -0,0 +1,105 @@
+/*=========================================================================
+ *
+ *  Copyright Leiden University Medical Center, Erasmus University Medical
+ *  Center and contributors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#ifndef selxIdentityTransformRegistrationComponent_h
+#define selxIdentityTransformRegistrationComponent_h
+
+#include "selxSuperElastixComponent.h"
+#include "selxSinksAndSourcesInterfaces.h"
+#include "selxItkObjectInterfaces.h"
+
+#include "itkImageSource.h"
+
+namespace selx
+{
+template< int Dimensionality, class TPixel >
+class IdentityTransformRegistrationComponent :
+  public SuperElastixComponent<
+  Accepting<
+  itkImageFixedInterface< Dimensionality, TPixel >,
+  itkImageMovingInterface< Dimensionality, TPixel >
+  >,
+  Providing< RunRegistrationInterface,
+  DisplacementFieldItkImageSourceInterface< Dimensionality, TPixel >
+  >
+  >
+{
+public:
+
+  /** Standard ITK typedefs. */
+  typedef IdentityTransformRegistrationComponent<
+    Dimensionality, TPixel
+    >                                      Self;
+  typedef SuperElastixComponent<
+    Accepting<
+    itkImageFixedInterface< Dimensionality, TPixel >,
+    itkImageMovingInterface< Dimensionality, TPixel >
+    >,
+    Providing< RunRegistrationInterface,
+    DisplacementFieldItkImageSourceInterface< Dimensionality, TPixel >
+    >
+    > Superclass;
+  typedef std::shared_ptr< Self >       Pointer;
+  typedef std::shared_ptr< const Self > ConstPointer;
+
+  IdentityTransformRegistrationComponent( const std::string & name, LoggerImpl & logger );
+  virtual ~IdentityTransformRegistrationComponent();
+
+  typedef typename ComponentBase::CriterionType CriterionType;
+  typedef TPixel                                PixelType;
+
+  // fixed and moving image types are all the same, these aliases can be used to be explicit. 
+  using MovingImageType = typename itkImageMovingInterface< Dimensionality, TPixel >::ItkImageType;
```

Comment:
The interfaces starts with lower-case itk while everything else starts with Itk. We should harmonize the conventions at some point.

Reply:
added to issue #130